ext/node/polyfills/http.ts

```diff
@@ -547,8 +547,11 @@ export class IncomingMessageForServer extends NodeReadable {
   #req: Request;
   url: string;
   method: string;
+  // Mock socket object.
```

Comment:
Why are these called "mock" if you use actual connection address?

Reply:
`mock` might be a confusing word, but this `socket` is a `net.Socket` in Node.js, which is skipped here.